Description: Fix test fixture for test misc::list_alignment
 Commit 0eb2a067, "Release 1.28.0" added a space in that line, but somehow the
 code in our test isn't happy about it.
 The commit changed the number of spaces between " and # from 10 to 11, and
 this patch changes it back to 10.
Last-Update: 2024-06-13
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/tests/misc.rs
+++ b/tests/misc.rs
@@ -886,3 +886,3 @@
     Available recipes:
-        a Z="\t z"           # something else
+        a Z="\t z"          # something else
         hello a b='B	' c='C' # this does a thing
